docs/source/updates/updates.rst

```diff
@@ -334,6 +334,9 @@ There are several ``%language`` pragmas in Idris 1, which define various
 experimental extensions. None of these are available in Idris 2, although
 extensions may be defined in the future.
 
+Also removed as the ``%access`` pragma for default visibility, use visibly
+modifiers on each declaration instead.
+
```

Comment:
Thanks for the updates! There's a typo above though ("visibly" rather than "visibility").

Reply:
Sound be fixed with the latest push.